apps/documentation/public/polyfill-db.json

```diff
@@ -0,0 +1,11 @@
+{
+		"polyfills": [
+				{ 
+						"identifier": "HTMLElement",
+						"imports": ["@lit-labs/ssr-dom-shim", "HTMLElement"]
```

Comment:
I'm wondering if we should support a list of available polyfills for each element.
Example accounting for my previous comment:

```json
{
  "polyfills": [
    {
      "identifier": "HTMLElement",
      "imports": [
        {
          "package": "@lit-labs/ssr-dom-shim",
          "name": "HTMLElement"
        },
        {
          "package": "@anotherOne",
          "name": "HTMLElement"
        }
      ]
    }
  ]
}

```

Reply:
I think it gets hard to maintain. WDYT?

Reply:
Ok, let's keep a single import for each polyfill.